### PR TITLE
added realpath cache dashboard

### PR DESCRIPTION
**Pull request by Gamesh**

### CODE REVIEW

**Comment by JorgenEvens** [comment]
Thanks for this!
I will go through the PR this evening.

File: cache.php [hunk 1]
```diff
@@ -95,6 +95,26 @@ function human_size( $s ) {
...
 
+	function machine_size( $val ) {
+        $val = trim($val);
+        $last = strtolower($val[strlen($val)-1]);
+
+        if (!is_numeric($last)){
+            $val = (int) substr($val, 0, -1);
+        }
+
+        switch($last) {
```
Comment (JorgenEvens) [inline cache.php]: Would you mind changing this into a few `if` statements instead of the `switch`?

**Comment by JorgenEvens** [comment]
Thanks for the PR, I didn't even knew this information was available!

It's now included in the [latest release](https://github.com/JorgenEvens/php-cache-dashboard/releases/tag/v1.1.0). 🎉

**Comment by Gamesh** [comment]
imho the switch statement looked cleaner :) it's best suited for cases like this where you need to execute code on multiple cases, but as you wish

thanks for merging

**Comment by JorgenEvens** [comment]
I felt the switch statement fall-through was not obvious enough and could cause me issues in the future. The if statement is simply more explicit.

**Pull request merged**